### PR TITLE
enable pv for elasticsearch for testing

### PR DESCRIPTION
**Pull request by richm**
We should enable using a pv for ES and ES OPS for testing since that is what customers will do.

### CODE REVIEW

**Comment by richm** [comment]
[test]

**Comment by richm** [comment]
aos-ci-test

**Comment by richm** [comment]
aos-ci-test

**Comment by richm** [comment]
aos-ci-test

**Comment by richm** [comment]
aos-ci-test

**Comment by richm** [comment]
aos-ci-test

**Comment by richm** [comment]
aos-ci-test

**Comment by richm** [comment]
aos-ci-test

**Comment by richm** [comment]
aos-ci-test

**Comment by richm** [comment]
aos-ci-test

**Comment by richm** [comment]
aos-ci-test

**Comment by richm** [comment]
aos-ci-test

**Comment by richm** [comment]
aos-ci-test

**Comment by richm** [comment]
aos-ci-test

**Comment by richm** [comment]
aos-ci-test

**Comment by richm** [comment]
aos-ci-test

**Comment by richm** [comment]
aos-ci-test

**Comment by richm** [comment]
aos-ci-test

**Comment by richm** [comment]
[test]

**Comment by richm** [comment]
aos-ci-test

File: hack/testing/logging.sh [hunk 1]
```diff
@@ -33,6 +33,8 @@ USE_LOGGING_DEPLOYER_SCRIPT=
 ENABLE_OPS_CLUSTER=${ENABLE_OPS_CLUSTER:-false}
 DEBUG_FAILURES=${DEBUG_FAILURES:-false}
 USE_LOCAL_SOURCE=${USE_LOCAL_SOURCE:-false}
+ES_VOLUME=${ES_VOLUME:-/var/lib/es}
+ES_OPS_VOLUME=${ES_OPS_VOLUME:-/var/lib/es-ops}
```
Comment (ewolinetz) [inline hack/testing/logging.sh]: we can guarantee that this will go away with the AMI, correct?

**Comment by ewolinetz** [comment]
other than question LGTM

File: hack/testing/logging.sh (hunk 1 above)
Comment (richm) [inline hack/testing/logging.sh]: Yes.  We're just using the local disk on the (virtual) machine.  When the machine goes away, so will the disk.  We could eventually mount a real persistent volume, wipe out the machine, create a new machine with the same pv, and see if es still works.

**Pull request merged**